### PR TITLE
Refactor: Refactor block and block feedback URLs

### LINKED ISSUE #1180: Update URLS & Slugs in both Backend & Frontend in one PR (see also below)

### PR DESCRIPTION
**Pull request by drikusroor**
This pull request refactors the URLs for blocks and block feedback in the codebase. It adds the "block" prefix from the URLs and removes the "collection" prefix from experiment URLs. This improves the consistency and readability of the URLs.

## API

- `/{blockSlug` => `/block/{blockSlug}`
- `/{blockSlug/feedback` => `/block/{feedback}`
- `/collection/{expCollectionSlug` => `/{expCollectionSlug}`

Resolves #1180

### CODE REVIEW

File: backend/experiment/tests/test_admin_experiment.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ class MockRequest:
 request = MockRequest()
 
 
-class TestAdminExperiment(TestCase):
+class TestAdminBlcok(TestCase):
```
Comment (BeritJanssen) [inline backend/experiment/tests/test_admin_experiment.py:30]:
```suggestion
class TestAdminBlock(TestCase):
```

File: frontend/src/components/App/App.tsx [hunk 2]
```diff
@@ -69,7 +69,7 @@ const App = () => {
                         {/* Default experiment */}
                         <Route path="/" exact>
                             <Redirect
-                                to={URLS.block.replace(":slug", EXPERIMENT_SLUG)}
+                                to={URLS.experimentCollection.replace(":slug", EXPERIMENT_SLUG)}
```
Comment (BeritJanssen) [inline frontend/src/components/App/App.tsx:72]: Isn't `EXPERIMENT_SLUG` related to what's now called a `Block`?

File: frontend/src/components/Question/__snapshots__/Question.test.tsx.snap
Comment (BeritJanssen) [inline frontend/src/components/Question/__snapshots__/Question.test.tsx.snap:1]: Should this snapshot be committed?

**Review by BeritJanssen — CHANGES_REQUESTED** [review]
Nice! But perhaps do remove & .gitignore the `__snapshots__` directories?

File: frontend/src/components/App/App.tsx (hunk 2 above)
Comment (drikusroor) [reply frontend/src/components/App/App.tsx:72]: I figured the default experiment slug as set in the environment variables will now actually be the slug of an experiment(collection) like `toontjehoger`, or what do you think?

Comment (drikusroor) [reply frontend/src/components/App/App.tsx:72]: I'm merging this. I could do something with `EXPERIMENT_SLUG` in a later PR if needed :-)

**Pull request merged**